ui/actions/nuxeo-action-button-styles.js

```diff
@@ -42,6 +42,30 @@ const template = html`
           cursor: pointer;
           @apply --nuxeo-action-button;
         }
+        paper-button.primary {
```

Comment:
If I'm seeing it correctly, this file is related to `action buttons` and probably the aim was to just encapsulate/factor their styles.
We probably don't want or need this `paper-button` styles here. 🤔 🤷‍♂️

Reply:
for some reason, the global paper-button is not taking effect, so either I declare it here or I need to repeat it for every nuxe-action-button

Reply:
i made a new commit making a specific style for the button